v3/src/components/graph/adornments/movable-line.tsx

```diff
@@ -56,6 +56,23 @@
     xSubAxesCount = layout.getAxisMultiScale('bottom')?.repetitions ?? 1,
     ySubAxesCount = layout.getAxisMultiScale('left')?.repetitions ?? 1
 
+  const setEquationContainerClass = useCallback(() => {
```

Comment:
I'm confused about the naming. Nothing is being set, right? Why not just name it equationContainerClass or getEquationContainerClass?

Reply:
@bfinzer I think it is setting something in that it's building a value out of other values and according to a certain condition. I think `get` would imply the function fetches a value from another source.

I think we generally append `set` (or `get`) to function names so they don't clash with associated variable names. If the function was named `equationContainerClass`, you couldn't do `const equationContainerClass = equationContainerClass()`.

I suppose we could have the function named `equationContainerClass` and forgo the use of the variable. In other words, use `equationContainerClass()` where we're now using `equationContainerClass`, but I'm not sure that's a pattern we generally follow?

In any case, I'm not super happy with these functions in general. Maybe there's a more concise way to target the elements without having to build these somewhat complex selector values. I wasn't able to come up with an alternative previously, but I could give it another try if you think it's worth doing?

Reply:
I think using equationContainerClass() in place of the variable is a good solution even though it's somewhat unusual.